src/utils/__tests__/date.test.ts

```diff
@@ -1,4 +1,5 @@
-import { isNone, isSome } from "fp-ts/lib/Option";
+import * as E from "fp-ts/Either"
```

Comment:
ditto

Reply:
Sorry, I do not understand the advise here